src/components/RoarDataTable.vue

```diff
@@ -304,7 +330,7 @@ const props = defineProps({
   loading: { type: Boolean, default: false },
   lazy: { type: Boolean, default: false },
   lazyPreSorting: { type: Array, required: false, default: () => [] },
-  allowFiltering: { type: Boolean, required: false, default: true },
+  resetFilters: { type: Function, required: false, default: () => {} },
```

Comment:
Can I suggest switching from passing a function, and emit an event with the relevant info instead? This data flow keeps consistency with the rest of the app and is Vue's preferred way of handling data flow between parent / child components. I completely understand using this for testing though for debugging purposes! 
This would be easily switched by using `const emits = defineEmits(['resetFilters'])`, `emit('resetFilters')`, where we call RoarDataTable using `@reset-filters="functionName"`

Reply:
Thanks for the recommendation! I've updated the function call to an emit.